tests/manage/z_cluster/test_validate_ceph_config_values.py

```diff
@@ -0,0 +1,50 @@
+import logging
+import pytest
+import collections
+
+
+from ocs_ci.framework.testlib import ManageTest, tier1
+from ocs_ci.framework import config
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs import constants, defaults
+
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@pytest.mark.skipif(
+    config.DEPLOYMENT["ceph_debug"],
```

Comment:
```suggestion
    config.DEPLOYMENT.get("ceph_debug"),
```

Reply:
@petr-balogh Default value of `ceph_debug` is present in defaults, so it should not matter. Or do I miss something?

Reply:
done